cli/src/tests/query_test.rs

```diff
@@ -737,6 +737,57 @@ fn test_query_matches_capturing_error_nodes() {
     });
 }
 
+#[test]
+fn test_query_matches_capturing_missing_nodes() {
+    allocations::record(|| {
+        let language = get_language("javascript");
+        let query = Query::new(
+            &language,
+            "
+            (MISSING) @missing-node
+            ",
+        )
+        .unwrap();
```

Comment:
single line
```suggestion
        let query = Query::new(
            &language,
            "(MISSING) @missing-node",
        )
        .unwrap();
```

rustfmt might make the whole call on one line then